WireAPI/Sources/WireAPI/APIs/ConversationsAPI/ConversationsAPIV5.swift

```diff
@@ -20,4 +20,135 @@ import Foundation
+            failed: failed
+        )
+    }
+}
+
+// MARK: -
+
+private struct ConversationV5: Decodable, ToAPIModelConvertible {
+    enum CodingKeys: String, CodingKey {
+        case access
+        case accessRole = "access_role"
+        case cipherSuite = "cipher_suite"
+        case creator
+        case epoch
+        case epochTimestamp = "epoch_timestamp"
+        case id
+        case lastEvent = "last_event"
+        case lastEventTime = "last_event_time"
+        case members
+        case messageProtocol = "protocol"
+        case messageTimer = "message_timer"
+        case mlsGroupID = "group_id"
+        case name
+        case qualifiedID = "qualified_id"
+        case readReceiptMode = "receipt_mode"
+        case teamID = "team"
+        case type
+    }
+
+    var access: [String]?
+    var accessRoles: [String]?
+    var cipherSuite: UInt16? // New field
+    var creator: UUID?
+    var epoch: UInt?
+    var epochTimestamp: Date? // New field
+    var id: UUID?
+    var lastEvent: String?
+    var lastEventTime: String?
+    var members: QualifiedConversationMembers?
+    var messageProtocol: String?
+    var messageTimer: TimeInterval?
+    var mlsGroupID: String?
+    var name: String?
+    var qualifiedID: QualifiedID?
+    var readReceiptMode: Int?
+    var teamID: UUID?
+    var type: Int?
+
+    init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+
+        access = try container.decodeIfPresent([String].self, forKey: .access)
+        accessRoles = try container.decodeIfPresent([String].self, forKey: .accessRole)
+        cipherSuite = try container.decodeIfPresent(UInt16.self, forKey: .cipherSuite)
+        creator = try container.decodeIfPresent(UUID.self, forKey: .creator)
+        epoch = try container.decodeIfPresent(UInt.self, forKey: .epoch)
+        epochTimestamp = try container.decodeIfPresent(Date.self, forKey: .epochTimestamp)
+        id = try container.decodeIfPresent(UUID.self, forKey: .id)
+        lastEvent = try container.decodeIfPresent(String.self, forKey: .lastEvent)
+        lastEventTime = try container.decodeIfPresent(String.self, forKey: .lastEventTime)
+        members = try container.decodeIfPresent(QualifiedConversationMembers.self, forKey: .members)
+        messageProtocol = try container.decodeIfPresent(String.self, forKey: .messageProtocol)
+        messageTimer = try container.decodeIfPresent(TimeInterval.self, forKey: .messageTimer)
+        mlsGroupID = try container.decodeIfPresent(String.self, forKey: .mlsGroupID)
+        name = try container.decodeIfPresent(String.self, forKey: .name)
+        qualifiedID = try container.decodeIfPresent(QualifiedID.self, forKey: .qualifiedID)
+        readReceiptMode = try container.decodeIfPresent(Int.self, forKey: .readReceiptMode)
+        teamID = try container.decodeIfPresent(UUID.self, forKey: .teamID)
+        type = try container.decodeIfPresent(Int.self, forKey: .type)
+    }
```

Comment:
Is this still necessary?

Reply:
No, thanks removed 9fe392f45956b5508304aa87df73e39e47d79463